documentation/code.md

```diff
@@ -222,7 +222,7 @@ follow them, they could be incorporated together with new changes.
   is, the hierarchy of text files needs to be fleshed out sufficiently
   so the new state can be communicated to a person unfamiliar with the
   package. Examples:
-    - The scope of the solution a package implements changes as a
+    - The scope of the solution a package implements changes, as a
```

Comment:
I'm pretty sure this is wrong.

Reply:
Ha, the only reason I added the comma was for consistency; the next item has it too. I will instead remove it from both :)

Reply:
Done.